controller/DownloadSampleTemplate.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2014 (original work) Open Assessment Technologies SA
+ *
+ */
+
+namespace oat\taoQtiItem\controller;
+
+use oat\taoQtiItem\model\import\Repository\CsvTemplateRepository;
+use oat\taoQtiItem\model\import\Repository\MetadataRepository;
+use oat\taoQtiItem\model\import\Repository\TemplateRepositoryInterface;
+use oat\taoQtiItem\model\import\Parser\TemplateHeaderParser;
+use oat\taoQtiItem\model\import\Factory\CsvTemplateSampleLineFactory;
+use tao_actions_CommonModule;
+use common_exception_Error;
+use common_exception_MissingParameter;
+use SPLTempFileObject;
+
+/**
+ * DownloadSampleCsv Controller provide actions to download sample CSV
+ *
+ * @author Chinnu Francis - TAO Team - {@link http://www.tao.lu}
+ * @package taoQtiItem
+
+ * @license GPLv2  http://www.opensource.org/licenses/gpl-2.0.php
+ */
+class DownloadSampleTemplate extends tao_actions_CommonModule
```

Comment:
Do you think it is well defined in the controller "name and namespace" that it is about download a sample for item import? Do you think it could misslead to download a QTI template or other something else?

```suggestion
class DownloadSampleTemplate extends tao_actions_CommonModule
```

Reply:
It was done